special-use/special-use-data-schema.json

```diff
@@ -1,64 +1,87 @@
 {
     "$schema": "http://json-schema.org/draft-04/schema#",
     "title": "Special use application",
-    "description": "An application for a special use permit from the U.S. Forest Service",
+    "description": "A rough sketch of the POST JSON HTTP request for a special use permit from our ePermitting system the U.S. Forest Service",
```

Comment:
`from our ePermitting system the U.S. Forest Service` maybe the `from the U.S. Forest Service ePermitting System`.